pinot-controller/src/main/resources/app/pages/InstanceListingPage.tsx

```diff
@@ -38,13 +39,12 @@ const InstanceListingPage = () => {
   const classes = useStyles();
 
   const [fetching, setFetching] = useState(true);
-  const [instances, setInstances] = useState<DataTable>();
+  // const [instances, setInstances] = useState<DataTable>();
   const [clusterName, setClusterName] = useState('');
 
   const fetchData = async () => {
-    const instanceResponse = await PinotMethodUtils.getAllInstances();
-    const instanceType = startCase(window.location.hash.split('/')[1].slice(0, -1));
-    setInstances(pick(instanceResponse, instanceType));
+    // const instanceResponse = await PinotMethodUtils.getAllInstances();
+    // setInstances(pick(instanceResponse, instanceType));
```

Comment:
remove unwanted comments

Reply:
good catch. though i caught all of these